source/documentation/administration_guide/topics/Parameters_for_event_notifications_in_ovirt-engine-notifier.adoc

```diff
@@ -22,7 +22,7 @@ The event notifier configuration file can be found in */usr/share/ovirt-engine/s
 |DAYS_TO_KEEP_HISTORY |0 |This variable sets the number of days dispatched events will be preserved in the history table. If this variable is not set, events remain on the history table indefinitely.
 |FAILED_QUERIES_NOTIFICATION_THRESHOLD |30 |The number of failed queries after which a notification email is sent. A notification email is sent after the first failure to fetch notifications, and then once every time the number of failures specified by this variable is reached. If you specify a value of `0` or `1`, an email will be sent with each failure.
 |FAILED_QUERIES_NOTIFICATION_RECIPIENTS |none |The email addresses of the recipients to which notification emails will be sent. Email addresses must be separated by a comma. This entry has been deprecated by the `FILTER` variable.
-|DAYS_TO_SEND_ON_STARTUP |0 |The number of days of old events that will be processed and sent when the notifier starts.
+|DAYS_TO_SEND_ON_STARTUP |0 |The number of days of old events that will be processed and sent when the notifier starts. If the value is 0 and the service stops and starts after a while, all notifications between service stop and service start time will be lost, please set this value for 1 or a greater value if you want to get notifications on events that occured between service stop and start time.
```

Comment:
I suggest this rewrite so that it conforms with our documentation conventions:

"If the value is `0` and the service stops and subsequently starts, all notifications for events that occurred while the service was stopped are lost. You must set the value to `1` or greater if you want to receive these notifications."